web/src/main/webapp/WEB-INF/jsp/member/create.jsp

```diff
@@ -0,0 +1,22 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="x"%>
+<x:base title="Register new Member">
+    <jsp:attribute name="content">
+        <h1>Register new member</h1>
+        <form:form method="POST" modelAttribute="createMember">
+            <div class="form-group">
+                <form:input path="member.givenName" type="text" placeholder="*Username" cssClass="form-control" />
```

Comment:
Use labels, not placeholders. Labels don't disappear after you started typing and work in all browsers